MrTheGood/projectb_android
# Mapview

Changes:
 * Added this amazing feature
 * Fixed bug where thing happened instead of thing
 * Improved this feature

How to test: 
 1. Do this
open the app and see the pointers
 2. Then do this
look at it

Screenshots:
If applicable, add a screenshot or video here to show the new feature
